lib/galaxy_test/selenium/test_uploads.py

```diff
@@ -90,7 +90,7 @@ def test_upload_pair(self):
     @selenium_test
     def test_upload_pair_specify_extension(self):
         self.upload_list([self.get_filename("1.tabular"), self.get_filename("2.tabular")], name="Test Pair", ext="txt", hide_source_items=False)
```

Comment:
If hide_source_items is False, this needs to be 3 right?

Is hide_source_items on ``upload_list`` broken maybe?

Reply:
No - I guess you're right. If you don't hide the source items you want to copy them... so it should be 5. You see - this is why you can't trust me to answer this question 😆

Reply:
:smile: it's confusing and counter-intuitive, even though it makes sense when you think about it.